### PR TITLE
ReadableStream constructor underlying source is only technically optional

### PR DESCRIPTION
**Pull request by hamishwillee**
You can create a ReadableStream with no source, but it isn't useful for anything since there is no way to get data into it.

### CODE REVIEW

File: files/en-us/web/api/readablestream/readablestream/index.md [hunk 1]
```diff
@@ -11,8 +11,9 @@ browser-compat: api.ReadableStream.ReadableStream
 ---
 {{APIRef("Streams")}}
 
-The **`ReadableStream()`** constructor creates
-and returns a readable stream object from the given handlers.
+The **`ReadableStream()`** constructor creates and returns a readable stream object from the given handlers.
+
+Note that while parameters are technically optional, omitting the `underlyingSource` will result in a stream that has no source, and that can't be read from (readers return a promise that will never be resolved).
```
Comment (wbamberg) [inline files/en-us/web/api/readablestream/readablestream/index.md:16]:
```suggestion
Note that while all parameters are technically optional, omitting the `underlyingSource` will result in a stream that has no source, and that can't be read from (readers return a promise that will never be resolved).
```

**Comment by hamishwillee** [comment]
Thanks @wbamberg - I merged your suggestion

**Comment by hamishwillee** [comment]
@teoli2003 Do you think you might look at this one too? Been sitting around a while, and is pretty trivial. Already fixed the minor comment by Will

**Review by teoli2003 — APPROVED** [review]
All fine!

**Pull request merged**